### ISSUE TITLE
Verify behavior of GRANT/REVOKE during database creation

### ISSUE THREAD

**Issue by beckettsean** [influxdb/influxdb.com#37]
https://github.com/influxdb/influxdb/pull/2763#discussion-diff-31739854

**Comment by gunnaraasen** [comment]
Behavior of `GRANT` and `REVOKE will be fixed once https://github.com/influxdb/influxdb/pull/3244 is merged.

**Comment by beckettsean** [comment]
https://github.com/influxdb/influxdb/pull/3244 is merged into 0.9.2, closing this issue.
